src/astro_databricks/operators/notebook.py

```diff
@@ -300,7 +300,7 @@ def launch_notebook_job(self):
         else:
             raise ValueError("Must specify either existing_cluster_id or new_cluster")
         runs_api = RunsApi(api_client)
-        run = runs_api.submit_run(run_json)
+        run = runs_api.submit_run(run_json, version=JOBS_API_VERSION)
```

Comment:
Do we need to worry about the consistency of API across databricks API version or should be ok since default version we are not changing?

Reply:
just wondering if we need to support all Databricks API versions, can we just support the latest one?

Reply:
we do support both 2.0 and 2.1. This works for both 2.0 and 2.1

Reply:
and the API version is configurable with an env variable set in the constant JOBS_API_VERSION